qcodes/instrument_drivers/Newport/AG_UC8.py

```diff
@@ -171,10 +244,20 @@ class Newport_AG_UC8(VisaInstrument):
             for example "ASRL3" for COM3.
     """
 
+    # By default, expect response to command within 1 second.
     default_timeout = 1.0
+
+    # Position scan commands (and absolute move commands) can take
+    # up to 2 minutes to complete.
     position_scan_timeout = 120.0
-    command_delay = 0.002   # wait 2 ms between commands
-    reset_delay = 0.05      # wait 50 ms after reset command
+
+    # After a command which does not generate a response, a short
+    # delay is needed before we can send the following command.
+    command_delay = 0.002
+
+    # After a reset command, a longer delay is needed before
+    # we can send the following command.
+    reset_delay = 0.05
```

Comment:
these four attributes, do you foresee that they will need to be tweaked by users? if so, it might make sense to make them python properties (so that there is a docstring) or even qcodes parameters (with `set_cmd=None, get_cmd=None` and `initial_value=<these numbers>`). and also please explain them in the example notebook, so that the users are aware of these communication-related details.

Reply:
These are not tunable parameters.
The values deal with the communication behaviour of the device. I would hope that one set of values covers all possible cases. I can not exclude the possibility that the values will have to be tweaked for some combination of USB port and device in the future, but I think such cases should be treated as a bug fix instead of a setup-specific tuning.

Reply:
ok.